operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -158,6 +158,16 @@ public String preDescription(String keySecretName, String certSecretName) {
             public String postDescription(String keySecretName, String certSecretName) {
                 return "CA key (in " + keySecretName + ") replaced";
             }
+        },
+        REGENERATED_CERT_INFO() {
```

Comment:
As above, I do not think INFO is the right word. Either be specific that DNS names changed ur find something more suitable.